test/controllers/APIControllerSpec.scala

```diff
@@ -67,7 +68,22 @@ class APIControllerSpec extends SpecBase {
       val result = controller.getSummaryDataForVRN("123456789")(fakeRequest)
       status(result) shouldBe Status.OK
       val apiDataToReturn: APIModel = APIModel(
-        mockETMPPayloadForAPIResponseData.pointsTotal
+        mockETMPPayloadForAPIResponseData.pointsTotal,
+        mockETMPPayloadForAPIResponseData.latePaymentPenalties.map(_.count(_.status==PointStatusEnum.Estimated)).getOrElse(0)
+          + mockETMPPayloadForAPIResponseData.penaltyPoints.count(_.status==PointStatusEnum.Estimated)
```

Comment:
As this involves some manipulation, could you put the exist amount you expect rather than working it out? Like so
```suggestion
        noOfEstimatedPenalties = 2
```